src/services/todos.js

```diff
@@ -0,0 +1,21 @@
+import { client, checkError } from './client';
+
+export async function getTodos() {
```

Comment:
Adding ordering to your fetch call will keep the chores in the order they were added.